src/App.tsx

```diff
@@ -1,16 +1,21 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [moviesList, setMoviesList] = useState([...moviesFromServer]);
```

Comment:
```suggestion
  const [moviesList, setMoviesList] = useState(moviesFromServer);
```
`moviesFromServer` is a constant, so can be used like so